### DISCUSSION TITLE
Windows Terminal v1.13.1143

### DISCUSSION THREAD

**Discussion by DHowett** [microsoft/terminal#13156] [Announcements]
This release brings many of the preview changes in Windows Terminal 1.13 to the Stable channel. Notably:

> **IMPORTANT**
> This version was made available to the Dev External flighting ring (Windows Insiders) first, and will be
> released to general availability one or two weeks later depending on its reliability.

* You can now configure a profile to automatically launch as Administrator.
* There is a new action, "Restore last closed pane or tab," that will do _roughly_ what it says on  the tin.
* You can now change the bell sound with the `profile.bellSound` setting
* Terminal has learned to save and restore your last opened window, position and all! Check it out in Settings > Startup.

Note that the new text rendering engine is not included in this Stable build.

Please see the following release notes for additional details:

* [Windows Terminal Preview v1.13.1098](https://github.com/microsoft/terminal/releases/tag/v1.13.10983.0)
* [Windows Terminal Preview v1.13.1073](https://github.com/microsoft/terminal/releases/tag/v1.13.10733.0)
* [Windows Terminal Preview v1.13.10395.0](https://github.com/microsoft/terminal/releases/tag/v1.13.10395.0)
* [Windows Terminal Preview v1.13.10336.0](https://github.com/microsoft/terminal/releases/tag/v1.13.10336.0)

As a reminder, Terminal 1.12 is the last version of Windows Terminal that supports Windows 19H1 or 19H2.
That version of windows is going out of support soon, so you may want to consider upgrading.

<details>
<summary>Preinstallation Kit info</summary>

A [preinstallation](https://docs.microsoft.com/en-us/windows/msix/desktop/deploy-preinstalled-apps) kit is available for system integrators and OEMs interested in prepackaging Windows Terminal with a Windows image. More information is available in the [DISM documentation on preinstallation](https://docs.microsoft.com/windows-hardware/manufacture/desktop/preinstall-apps-using-dism). Users who do not intend to preinstall Windows Terminal should continue using the _msixbundle_ distribution.

</details>

<details>
<summary>Why are there so many packages? How do I choose?</summary>
This version of Windows Terminal is distributed in two bundles, one of which works on Windows 10-11 and the other of
which only works on Windows 11. The Windows 11 version is much smaller because we no longer need to work around a
platform issue related to our dependencies.

If you intend on using Terminal as an unpackaged application--that is, extracting the `msix` file--we recommend that
you use the `Win10` bundle. You will need the [Visual C++ runtime redistributable](https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist).

In addition, if you install the _packaged_ version on either Windows 10 or Windows 11, it now depends on [the Visual C++ Universal Runtime Package](https://docs.microsoft.com/en-us/troubleshoot/developer/visualstudio/cpp/libraries/c-runtime-packages-desktop-bridge?msclkid=3e1dd82aac8b11ec8f9dbd86d5979ca8#how-to-install-and-update-desktop-framework-packages).

Despite these distributions having different version numbers, they are built from the same code and there is no
functional difference between them.

If you install the Windows 10 verison on Windows 11, it will probably automatically upgrade itself to the Windows 11
version. It turns out that it is impossible to have two bundles with _the same version number_, so it has to be this
way.
</details>

Also included in this release are some bug fixes and changed backported from 1.14:

# Changes
## UI
* @dansmor7 contributed some visual changes to the tabs, scrollbar, new tab button, caption buttons, color picker, settings UI, command palette, and search box to move us closer to the Windows 11 design language. Really just about any WinUI surface we have, it's been polished up! (thanks @dansmor7!) (#12913) (#12916) (#12973) (#13083) 
* `RadioButton`s in the settings UI have been replaced with `ComboBox`es. This gives an added bonus to keyboard and screen reader users, and makes it easier to navigate through and change these settings. (#12833)  

## Interactivity
* Terminal is now aware of toggled state for  <kbd>Caps Lock</kbd>, <kbd>Scroll Lock</kbd>, and <kbd>Num Lock</kbd>  (#12823) (thanks @matkaas!) 

## Settings
* The `trimBlockSelection` global setting now defaults to `true` (#12737) 
* Terminal now ignores `newTab` actions with a profile index greater than the number of profiles (#11621) 
* `bellSound` is now in the schema (#13035) (thanks @pizzaz93!) 

# Bug Fixes
* Terminal should be able to find Cascadia Mono... Third time's the charm? 🍀 (#12904) 
* `commandline` in `profile.defaults` should no longer override the `commandline`s of profiles that specify cmd.exe or powershell.exe. (#12906) 
* Get rid of a memory leak in onecore interactivity (#12340)  
* Screen readers can now read some settings in the UI better (#13032) 
* Replace "acrylic" with "acrylic material" for localization purposes (#12505) 
* The "close tab" button color now matches the tab text color (#13018) (thanks @ianjoneill!) 

## Reliability
* Fix a crash when deleting the last profile in the settings UI (#13044) 

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/terminal/releases/tag/v1.13.11431.0'>Windows Terminal v1.13.1143</a>.</em>

**Comment by scottmuz7** [comment]
The windows 10 msixbundle failed for me (on Windows 10 20H2), with error "App Installer failed to install package dependencies. Ask the developer for  package."

I've installed the 1.12.10393.0 version and previously fine.

**Reply by scottmuz7** [reply]
All good now just needed to install Microsoft.VCLibs.140.00.UWPDesktop from the preinstall kit.

**Comment by Kralizek** [comment]
How long before the update pops up in the Store or on winget?

**Reply by glawrence** [reply]
I have just updated via WinGet but it seems that the update via Store is very slow to appear, again

**Comment by nisancigokmen** [comment]
wow
